setup/release.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+python ./setup/setup_idaplugin.py sdist --dist-dir ./setup/idaplugin_dist --formats=zip upload -r pypitest
+python ./setup/setup_server.py sdist --dist-dir ./setup/server_dist --formats=zip upload -r pypitest
+
+# cleanup
+rm -r ./setup/idaplugin_dist ./setup/server_dist ./rematch_idaplugin.egg-info ./server.egg-info
```

Comment:
could you fix those paths? ./setup is the directory we're present in. same for idaplugin.
the ```rm``` should only happen if the directory doesn't exist.

Also, I think it would be better to write it in python rather than bash so it would obsolete rather than having
two versions, (Windows + Linux/OSX/etc).

I think the package name should match the egg you're creating (in ```server.egg-info```).